test/e2e/dra/deploy.go

```diff
@@ -65,8 +65,8 @@ import (
 )
 
 const (
-	NodePrepareResourcesMethod   = "/v1beta1.DRAPlugin/NodePrepareResources"
-	NodeUnprepareResourcesMethod = "/v1beta1.DRAPlugin/NodeUnprepareResources"
+	NodePrepareResourcesMethod   = "/k8s.io.kubelet.pkg.apis.dra.v1beta1.DRAPlugin/NodePrepareResources"
+	NodeUnprepareResourcesMethod = "/k8s.io.kubelet.pkg.apis.dra.v1beta1.DRAPlugin/NodeUnprepareResources"
```

Comment:
Changing the method name feels like a breaking change. While working on https://github.com/kubernetes/kubernetes/pull/128659, I saw that we `cd` into the api.proto path before generating the definitions: https://github.com/kubernetes/kubernetes/blob/f59dd4bce33f454e5c9386be56fa3d11e3bb2d7a/hack/lib/protoc.sh#L63

If we remove that part, like:
```patch
diff --git a/hack/lib/protoc.sh b/hack/lib/protoc.sh
index 0812f76eaee..2272db43f31 100644
--- a/hack/lib/protoc.sh
+++ b/hack/lib/protoc.sh
@@ -59,21 +59,17 @@ function kube::protoc::protoc() {
   local package=${1}
   gogopath=$(dirname "$(kube::util::find-binary "protoc-gen-gogo")")

-  (
-    cd "${package}"
-
-    # This invocation of --gogo_out produces its output in the current
-    # directory (despite gogo docs saying it would be source-relative, it
-    # isn't).  The inputs to this function do not all have a common root, so
-    # this works best for all inputs.
-    PATH="${gogopath}:${PATH}" protoc \
-      --proto_path="$(pwd -P)" \
-      --proto_path="${KUBE_ROOT}/vendor" \
-      --proto_path="${KUBE_ROOT}/staging/src" \
-      --proto_path="${KUBE_ROOT}/third_party/protobuf" \
-      --gogo_out=paths=source_relative,plugins=grpc:. \
-      api.proto
-  )
+  # This invocation of --gogo_out produces its output in the current
+  # directory (despite gogo docs saying it would be source-relative, it
+  # isn't).  The inputs to this function do not all have a common root, so
+  # this works best for all inputs.
+  PATH="${gogopath}:${PATH}" protoc \
+    --proto_path="$(pwd -P)" \
+    --proto_path="${KUBE_ROOT}/vendor" \
+    --proto_path="${KUBE_ROOT}/staging/src" \
+    --proto_path="${KUBE_ROOT}/third_party/protobuf" \
+    --gogo_out=paths=source_relative,plugins=grpc:. \
+    "${package}/api.proto"
 }

 # Formats $1/api.pb.go, adds the boilerplate comments and run gofmt on it
```

And regenerate the profos (`./hack/update-codegen.sh protobindings`), then the diff for `staging/src/k8s.io/kubelet/pkg/apis/dra/v1beta1/api.pb.go` would look like this:

```patch
diff --git a/staging/src/k8s.io/kubelet/pkg/apis/dra/v1beta1/api.pb.go b/staging/src/k8s.io/kubelet/pkg/apis/dra/v1beta1/api.pb.go
index 2f024c614f1..b55ddfdae54 100644
--- a/staging/src/k8s.io/kubelet/pkg/apis/dra/v1beta1/api.pb.go
+++ b/staging/src/k8s.io/kubelet/pkg/apis/dra/v1beta1/api.pb.go
@@ -15,7 +15,7 @@ limitations under the License.
 */
 
 // Code generated by protoc-gen-gogo. DO NOT EDIT.
-// source: api.proto
+// source: staging/src/k8s.io/kubelet/pkg/apis/dra/v1beta1/api.proto
 
 package v1beta1
 
@@ -56,7 +56,7 @@ type NodePrepareResourcesRequest struct {
 func (m *NodePrepareResourcesRequest) Reset()      { *m = NodePrepareResourcesRequest{} }
 func (*NodePrepareResourcesRequest) ProtoMessage() {}
 func (*NodePrepareResourcesRequest) Descriptor() ([]byte, []int) {
-	return fileDescriptor_00212fb1f9d3bf1c, []int{0}
+	return fileDescriptor_50e01ff35787432a, []int{0}
 }
 func (m *NodePrepareResourcesRequest) XXX_Unmarshal(b []byte) error {
 	return m.Unmarshal(b)
@@ -107,7 +107,7 @@ type NodePrepareResourcesResponse struct {
 func (m *NodePrepareResourcesResponse) Reset()      { *m = NodePrepareResourcesResponse{} }
 func (*NodePrepareResourcesResponse) ProtoMessage() {}
 func (*NodePrepareResourcesResponse) Descriptor() ([]byte, []int) {
-	return fileDescriptor_00212fb1f9d3bf1c, []int{1}
+	return fileDescriptor_50e01ff35787432a, []int{1}
 }
 func (m *NodePrepareResourcesResponse) XXX_Unmarshal(b []byte) error {
 	return m.Unmarshal(b)
@@ -159,7 +159,7 @@ type NodePrepareResourceResponse struct {
 func (m *NodePrepareResourceResponse) Reset()      { *m = NodePrepareResourceResponse{} }
 func (*NodePrepareResourceResponse) ProtoMessage() {}
 func (*NodePrepareResourceResponse) Descriptor() ([]byte, []int) {
-	return fileDescriptor_00212fb1f9d3bf1c, []int{2}
+	return fileDescriptor_50e01ff35787432a, []int{2}
 }
 func (m *NodePrepareResourceResponse) XXX_Unmarshal(b []byte) error {
 	return m.Unmarshal(b)
@@ -220,7 +220,7 @@ type Device struct {
 func (m *Device) Reset()      { *m = Device{} }
 func (*Device) ProtoMessage() {}
 func (*Device) Descriptor() ([]byte, []int) {
-	return fileDescriptor_00212fb1f9d3bf1c, []int{3}
+	return fileDescriptor_50e01ff35787432a, []int{3}
 }
 func (m *Device) XXX_Unmarshal(b []byte) error {
 	return m.Unmarshal(b)
@@ -287,7 +287,7 @@ type NodeUnprepareResourcesRequest struct {
 func (m *NodeUnprepareResourcesRequest) Reset()      { *m = NodeUnprepareResourcesRequest{} }
 func (*NodeUnprepareResourcesRequest) ProtoMessage() {}
 func (*NodeUnprepareResourcesRequest) Descriptor() ([]byte, []int) {
-	return fileDescriptor_00212fb1f9d3bf1c, []int{4}
+	return fileDescriptor_50e01ff35787432a, []int{4}
 }
 func (m *NodeUnprepareResourcesRequest) XXX_Unmarshal(b []byte) error {
 	return m.Unmarshal(b)
@@ -335,7 +335,7 @@ type NodeUnprepareResourcesResponse struct {
 func (m *NodeUnprepareResourcesResponse) Reset()      { *m = NodeUnprepareResourcesResponse{} }
 func (*NodeUnprepareResourcesResponse) ProtoMessage() {}
 func (*NodeUnprepareResourcesResponse) Descriptor() ([]byte, []int) {
-	return fileDescriptor_00212fb1f9d3bf1c, []int{5}
+	return fileDescriptor_50e01ff35787432a, []int{5}
 }
 func (m *NodeUnprepareResourcesResponse) XXX_Unmarshal(b []byte) error {
 	return m.Unmarshal(b)
@@ -381,7 +381,7 @@ type NodeUnprepareResourceResponse struct {
 func (m *NodeUnprepareResourceResponse) Reset()      { *m = NodeUnprepareResourceResponse{} }
 func (*NodeUnprepareResourceResponse) ProtoMessage() {}
 func (*NodeUnprepareResourceResponse) Descriptor() ([]byte, []int) {
-	return fileDescriptor_00212fb1f9d3bf1c, []int{6}
+	return fileDescriptor_50e01ff35787432a, []int{6}
 }
 func (m *NodeUnprepareResourceResponse) XXX_Unmarshal(b []byte) error {
 	return m.Unmarshal(b)
@@ -434,7 +434,7 @@ type Claim struct {
 func (m *Claim) Reset()      { *m = Claim{} }
 func (*Claim) ProtoMessage() {}
 func (*Claim) Descriptor() ([]byte, []int) {
-	return fileDescriptor_00212fb1f9d3bf1c, []int{7}
+	return fileDescriptor_50e01ff35787432a, []int{7}
 }
 func (m *Claim) XXX_Unmarshal(b []byte) error {
 	return m.Unmarshal(b)
@@ -497,47 +497,50 @@ func init() {
 	proto.RegisterType((*Claim)(nil), "k8s.io.kubelet.pkg.apis.dra.v1beta1.Claim")
 }
 
-func init() { proto.RegisterFile("api.proto", fileDescriptor_00212fb1f9d3bf1c) }
-
-var fileDescriptor_00212fb1f9d3bf1c = []byte{
-	// 591 bytes of a gzipped FileDescriptorProto
-	0x1f, 0x8b, 0x08, 0x00, 0x00, 0x00, 0x00, 0x00, 0x02, 0xff, 0xb4, 0x55, 0x4d, 0x6f, 0xd3, 0x40,
-	0x10, 0xcd, 0x26, 0x6d, 0x8a, 0x27, 0x01, 0x55, 0xab, 0x0a, 0x85, 0xb4, 0x38, 0x91, 0x7b, 0x89,
-	0x40, 0xd8, 0x6a, 0x10, 0xa8, 0xe2, 0xd4, 0x38, 0xe9, 0x21, 0x07, 0x4a, 0x65, 0x51, 0x84, 0xb8,
-	0x54, 0x8e, 0xbd, 0x18, 0x2b, 0x1f, 0x6b, 0xbc, 0x76, 0xa4, 0x82, 0x90, 0x10, 0x27, 0x8e, 0xfc,
-	0x01, 0x4e, 0xfd, 0x33, 0x3d, 0xc2, 0x8d, 0x53, 0x45, 0xcd, 0x1f, 0x41, 0xde, 0xdd, 0x84, 0x50,
-	0x39, 0x10, 0x35, 0x70, 0xdb, 0x79, 0x3b, 0x33, 0xef, 0x79, 0xf6, 0x79, 0x17, 0x14, 0x3b, 0xf0,
-	0xf5, 0x20, 0xa4, 0x11, 0xc5, 0xdb, 0xfd, 0x5d, 0xa6, 0xfb, 0x54, 0xef, 0xc7, 0x3d, 0x32, 0x20,
-	0x91, 0x1e, 0xf4, 0x3d, 0xdd, 0x0e, 0x7c, 0xa6, 0xbb, 0xa1, 0xad, 0x8f, 0x77, 0x7a, 0x24, 0xb2,
-	0x77, 0xaa, 0xf7, 0x3c, 0x3f, 0x7a, 0x15, 0xf7, 0x74, 0x87, 0x0e, 0x0d, 0x8f, 0x7a, 0xd4, 0xe0,
-	0xb5, 0xbd, 0xf8, 0x25, 0x8f, 0x78, 0xc0, 0x57, 0xa2, 0xa7, 0x66, 0xc3, 0xe6, 0x01, 0x75, 0xc9,
-	0x61, 0x48, 0x02, 0x3b, 0x24, 0x16, 0x61, 0x34, 0x0e, 0x1d, 0xc2, 0x2c, 0xf2, 0x3a, 0x26, 0x2c,
-	0xc2, 0x26, 0x14, 0x9d, 0x81, 0xed, 0x0f, 0x59, 0x05, 0xd5, 0x0b, 0x8d, 0x52, 0xf3, 0x8e, 0xbe,
-	0x80, 0x06, 0xbd, 0x9d, 0x96, 0x58, 0xb2, 0x52, 0xfb, 0x90, 0x87, 0xad, 0x6c, 0x0e, 0x16, 0xd0,
-	0x11, 0x23, 0x98, 0x5c, 0x22, 0x79, 0xbc, 0x10, 0xc9, 0x9f, 0x5a, 0x0a, 0x05, 0x6c, 0x7f, 0x14,
-	0x85, 0x27, 0x13, 0x1d, 0xd5, 0xb7, 0x50, 0x9a, 0x81, 0xf1, 0x3a, 0x14, 0xfa, 0xe4, 0xa4, 0x82,
-	0xea, 0xa8, 0xa1, 0x58, 0xe9, 0x12, 0x3f, 0x83, 0xd5, 0xb1, 0x3d, 0x88, 0x49, 0x25, 0x5f, 0x47,
-	0x8d, 0x52, 0x73, 0xef, 0xaa, 0x32, 0x26, 0x2a, 0x2c, 0xd1, 0xee, 0x51, 0x7e, 0x17, 0x69, 0x6f,
-	0x32, 0xe7, 0x3c, 0x1d, 0xc1, 0x3e, 0xac, 0xb9, 0x64, 0xec, 0x3b, 0x64, 0x32, 0x83, 0xbb, 0x0b,
-	0x91, 0x77, 0x78, 0x8d, 0x35, 0xa9, 0xc5, 0x1b, 0xb0, 0x4a, 0xc2, 0x90, 0x86, 0xfc, 0x0b, 0x14,
-	0x4b, 0x04, 0xda, 0x29, 0x82, 0xa2, 0xc8, 0xc4, 0xdb, 0x70, 0x3d, 0x14, 0x47, 0x7b, 0x3c, 0xb2,
-	0x87, 0x92, 0x4d, 0xb1, 0xca, 0x12, 0x3c, 0x48, 0x31, 0xbc, 0x09, 0x4a, 0x40, 0xe9, 0x80, 0x67,
-	0xc8, 0x4e, 0xd7, 0x52, 0x20, 0xdd, 0xc5, 0x35, 0x28, 0x09, 0x36, 0xb1, 0x5d, 0xe0, 0xdb, 0x20,
-	0x20, 0x9e, 0xf0, 0x10, 0x6e, 0x38, 0xae, 0x7f, 0x2c, 0x93, 0x7c, 0x97, 0x55, 0x56, 0x52, 0x0e,
-	0x73, 0x3d, 0x39, 0xaf, 0x95, 0xdb, 0x9d, 0xae, 0x50, 0xd2, 0xed, 0x30, 0xab, 0xec, 0xb8, 0xbe,
-	0x8c, 0x5c, 0xa6, 0x39, 0x70, 0x3b, 0x9d, 0xd0, 0xd1, 0x28, 0xf8, 0x8f, 0x5e, 0xfc, 0x98, 0x07,
-	0x75, 0x1e, 0x8b, 0x3c, 0x0a, 0xef, 0x12, 0xcd, 0x93, 0x85, 0x6d, 0x30, 0xbf, 0x69, 0xa6, 0x1f,
-	0xdf, 0xfd, 0xcd, 0x8f, 0xcf, 0x7f, 0xf7, 0xa3, 0x79, 0x75, 0x21, 0x59, 0x8e, 0x7c, 0x30, 0x67,
-	0xde, 0xd3, 0x41, 0x4c, 0xcd, 0x84, 0x66, 0xcd, 0xf4, 0x14, 0x56, 0xb9, 0x6a, 0xbc, 0x05, 0x0a,
-	0xb7, 0x50, 0x60, 0x3b, 0x44, 0xa6, 0xfc, 0x02, 0xf0, 0x2d, 0x28, 0xc4, 0xbe, 0x2b, 0xdc, 0x63,
-	0xae, 0x25, 0xe7, 0xb5, 0xc2, 0x51, 0xb7, 0x63, 0xa5, 0x18, 0xc6, 0xb0, 0x32, 0x63, 0x1d, 0xbe,
-	0x6e, 0x7e, 0xcd, 0x83, 0xd2, 0xb1, 0x5a, 0x87, 0x83, 0xd8, 0xf3, 0x47, 0xf8, 0x33, 0x82, 0x8d,
-	0xac, 0xdf, 0x1b, 0xef, 0x2d, 0x71, 0x33, 0x70, 0x13, 0x55, 0x5b, 0x4b, 0xdf, 0x2d, 0x5a, 0x0e,
-	0x9f, 0x22, 0xb8, 0x99, 0x7d, 0xe0, 0xd8, 0x5c, 0xca, 0x2d, 0x42, 0x63, 0xfb, 0x1f, 0x38, 0x4e,
-	0xcb, 0x99, 0xad, 0xb3, 0x0b, 0x15, 0x7d, 0xbb, 0x50, 0x73, 0xef, 0x13, 0x15, 0x9d, 0x25, 0x2a,
-	0xfa, 0x92, 0xa8, 0xe8, 0x7b, 0xa2, 0xa2, 0x4f, 0x3f, 0xd4, 0xdc, 0x0b, 0xf9, 0x98, 0x18, 0x92,
-	0xc3, 0x08, 0xfa, 0x9e, 0x91, 0x72, 0x18, 0x6e, 0x68, 0x1b, 0x92, 0xa3, 0x57, 0xe4, 0x8f, 0xc4,
-	0xfd, 0x9f, 0x01, 0x00, 0x00, 0xff, 0xff, 0xda, 0x25, 0x38, 0x7d, 0x85, 0x06, 0x00, 0x00,
+func init() {
+	proto.RegisterFile("staging/src/k8s.io/kubelet/pkg/apis/dra/v1beta1/api.proto", fileDescriptor_50e01ff35787432a)
+}
+
+var fileDescriptor_50e01ff35787432a = []byte{
+	// 603 bytes of a gzipped FileDescriptorProto
+	0x1f, 0x8b, 0x08, 0x00, 0x00, 0x00, 0x00, 0x00, 0x02, 0xff, 0xb4, 0x55, 0xcd, 0x6e, 0xd3, 0x4c,
+	0x14, 0xcd, 0x24, 0xfd, 0xf9, 0x7c, 0xd3, 0x0f, 0x55, 0xa3, 0x0a, 0x85, 0xb6, 0x38, 0x91, 0xb3,
+	0x89, 0x40, 0xd8, 0x6a, 0x10, 0xa8, 0xb0, 0x6a, 0x9c, 0x74, 0x91, 0x05, 0xa5, 0xb2, 0x28, 0x42,
+	0x6c, 0x2a, 0xc7, 0x1e, 0x8c, 0x95, 0xc4, 0x63, 0x66, 0xec, 0x48, 0x05, 0x21, 0x21, 0x56, 0x2c,
+	0x79, 0x01, 0x56, 0x7d, 0x99, 0x2e, 0x61, 0xc7, 0xaa, 0xa2, 0xe6, 0x45, 0x90, 0x67, 0x9c, 0x10,
+	0x2a, 0x07, 0xa2, 0x06, 0x76, 0x33, 0x67, 0xce, 0xbd, 0xe7, 0xe4, 0xce, 0x89, 0x07, 0x1e, 0xf0,
+	0xc8, 0xf6, 0xfc, 0xc0, 0x33, 0x38, 0x73, 0x8c, 0xfe, 0x2e, 0xd7, 0x7d, 0x6a, 0xf4, 0xe3, 0x1e,
+	0x19, 0x90, 0xc8, 0x08, 0xfb, 0x9e, 0x61, 0x87, 0x3e, 0x37, 0x5c, 0x66, 0x1b, 0xa3, 0x9d, 0x1e,
+	0x89, 0xec, 0x9d, 0x14, 0xd0, 0x43, 0x46, 0x23, 0x8a, 0xeb, 0x92, 0xae, 0x67, 0x74, 0x3d, 0xec,
+	0x7b, 0x7a, 0x4a, 0xd7, 0x5d, 0x66, 0xeb, 0x19, 0x7d, 0xf3, 0x8e, 0xe7, 0x47, 0x2f, 0xe3, 0x9e,
+	0xee, 0xd0, 0xa1, 0xe1, 0x51, 0x8f, 0x1a, 0xa2, 0xb6, 0x17, 0xbf, 0x10, 0x3b, 0xb1, 0x11, 0x2b,
+	0xd9, 0x53, 0xb3, 0x61, 0xeb, 0x80, 0xba, 0xe4, 0x90, 0x91, 0xd0, 0x66, 0xc4, 0x22, 0x9c, 0xc6,
+	0xcc, 0x21, 0xdc, 0x22, 0xaf, 0x62, 0xc2, 0x23, 0x6c, 0xc2, 0x8a, 0x33, 0xb0, 0xfd, 0x21, 0xaf,
+	0xa0, 0x5a, 0xa9, 0x51, 0x6e, 0xde, 0xd2, 0xe7, 0xf0, 0xa0, 0xb7, 0xd3, 0x12, 0x2b, 0xab, 0xd4,
+	0xde, 0x17, 0x61, 0x3b, 0x5f, 0x83, 0x87, 0x34, 0xe0, 0x04, 0x93, 0x4b, 0x22, 0x8f, 0xe6, 0x12,
+	0xf9, 0x5d, 0x4b, 0xe9, 0x80, 0xef, 0x07, 0x11, 0x3b, 0x19, 0xfb, 0xd8, 0x7c, 0x03, 0xe5, 0x29,
+	0x18, 0xaf, 0x43, 0xa9, 0x4f, 0x4e, 0x2a, 0xa8, 0x86, 0x1a, 0x8a, 0x95, 0x2e, 0xf1, 0x53, 0x58,
+	0x1e, 0xd9, 0x83, 0x98, 0x54, 0x8a, 0x35, 0xd4, 0x28, 0x37, 0xf7, 0xae, 0x6a, 0x63, 0xec, 0xc2,
+	0x92, 0xed, 0x1e, 0x16, 0x77, 0x91, 0xf6, 0x3a, 0x77, 0xce, 0x93, 0x11, 0xec, 0xc3, 0xaa, 0x4b,
+	0x46, 0xbe, 0x43, 0xc6, 0x33, 0xb8, 0x3d, 0x97, 0x78, 0x47, 0xd4, 0x58, 0xe3, 0x5a, 0xbc, 0x01,
+	0xcb, 0x84, 0x31, 0xca, 0xc4, 0x2f, 0x50, 0x2c, 0xb9, 0xd1, 0x4e, 0x11, 0xac, 0x48, 0x26, 0xae,
+	0xc3, 0xff, 0x4c, 0x5e, 0xed, 0x71, 0x60, 0x0f, 0x33, 0x35, 0xc5, 0x5a, 0xcb, 0xc0, 0x83, 0x14,
+	0xc3, 0x5b, 0xa0, 0x84, 0x94, 0x0e, 0x04, 0x23, 0xeb, 0xf4, 0x5f, 0x0a, 0xa4, 0xa7, 0xb8, 0x0a,
+	0x65, 0xa9, 0x26, 0x8f, 0x4b, 0xe2, 0x18, 0x24, 0x24, 0x08, 0xf7, 0xe1, 0x9a, 0xe3, 0xfa, 0xc7,
+	0x19, 0xc9, 0x77, 0x79, 0x65, 0x29, 0xd5, 0x30, 0xd7, 0x93, 0xf3, 0xea, 0x5a, 0xbb, 0xd3, 0x95,
+	0x4e, 0xba, 0x1d, 0x6e, 0xad, 0x39, 0xae, 0x9f, 0xed, 0x5c, 0xae, 0x39, 0x70, 0x33, 0x9d, 0xd0,
+	0x51, 0x10, 0xfe, 0xc3, 0x2c, 0x7e, 0x28, 0x82, 0x3a, 0x4b, 0x25, 0xbb, 0x0a, 0xef, 0x92, 0xcc,
+	0xe3, 0xb9, 0x63, 0x30, 0xbb, 0x69, 0x6e, 0x1e, 0xdf, 0xfe, 0x29, 0x8f, 0xcf, 0x7e, 0xcd, 0xa3,
+	0x79, 0x75, 0x23, 0x79, 0x89, 0xbc, 0x37, 0x63, 0xde, 0x93, 0x41, 0x4c, 0xc2, 0x84, 0xa6, 0xc3,
+	0xf4, 0x04, 0x96, 0x85, 0x6b, 0xbc, 0x0d, 0x8a, 0x88, 0x50, 0x68, 0x3b, 0x24, 0xa3, 0xfc, 0x04,
+	0xf0, 0x0d, 0x28, 0xc5, 0xbe, 0x2b, 0xd3, 0x63, 0xae, 0x26, 0xe7, 0xd5, 0xd2, 0x51, 0xb7, 0x63,
+	0xa5, 0x18, 0xc6, 0xb0, 0x34, 0x15, 0x1d, 0xb1, 0x6e, 0x7e, 0x29, 0x82, 0xd2, 0xb1, 0x5a, 0x87,
+	0x83, 0xd8, 0xf3, 0x03, 0xfc, 0x09, 0xc1, 0x46, 0xde, 0xdf, 0x1b, 0xef, 0x2d, 0xf0, 0x65, 0x10,
+	0x21, 0xda, 0x6c, 0x2d, 0xfc, 0x6d, 0xd1, 0x0a, 0xf8, 0x14, 0xc1, 0xf5, 0xfc, 0x0b, 0xc7, 0xe6,
+	0x42, 0x69, 0x91, 0x1e, 0xdb, 0x7f, 0x21, 0x71, 0x5a, 0xc1, 0x6c, 0x9d, 0x5d, 0xa8, 0xe8, 0xeb,
+	0x85, 0x5a, 0x78, 0x97, 0xa8, 0xe8, 0x2c, 0x51, 0xd1, 0xe7, 0x44, 0x45, 0xdf, 0x12, 0x15, 0x7d,
+	0xfc, 0xae, 0x16, 0x9e, 0xd7, 0xe7, 0x78, 0x7b, 0x7a, 0x2b, 0xe2, 0x91, 0xb8, 0xfb, 0x23, 0x00,
+	0x00, 0xff, 0xff, 0x5d, 0xad, 0x05, 0xcd, 0xb5, 0x06, 0x00, 0x00,
 }
 
 // Reference imports to suppress errors if they are not otherwise used.
@@ -665,7 +668,7 @@ var _DRAPlugin_serviceDesc = grpc.ServiceDesc{
 		},
 	},
 	Streams:  []grpc.StreamDesc{},
-	Metadata: "api.proto",
+	Metadata: "staging/src/k8s.io/kubelet/pkg/apis/dra/v1beta1/api.proto",
 }
 
 func (m *NodePrepareResourcesRequest) Marshal() (dAtA []byte, err error) {
```